_posts/2019-10-03-dds-xrce.md

```diff
@@ -0,0 +1,10 @@
+---
+layout: post
+title:  "OMG DDS For Extremely Resource-Constrained Environments (DDS-XRCE)"
+categories: news
+author: The OpenDDS Team
+---
+
+The task of designing integrated solutions is frequently met with challenges when unusual architectural constraints from system and non-system sources are introduced into the equation. Normal integration patterns are desirable,but what happens when actors in your deployment introduce unique requirements?
+
+In [this SETT article]({{ site.data.oci.homepage }}/resources/publications/sett/october-2019-dds-for-extremely-resource-constrained-environments), Object Computing Solutions Architect, Marc Neeley, explores how today's industry standards are addressing complex integration challenges and provides a detailed review of the OMG specification, DDS for eXtremely Resource Constrained Environments (DDS-XRCE), which plays a significant role in extending solutions into environments with unique requirements.
```

Comment:
When the News is summarized on the front page, the first paragraph is shown followed by a "Continue reading" link.  I think we should move some of this to the first paragraph so that it's clear that this is a link to an article.